packages/api/tests/endpoints/test-granules.js

```diff
@@ -251,6 +262,8 @@ test.beforeEach(async (t) => {
         granule_id: granuleId1,
         status: 'completed',
         collection_cumulus_id: t.context.collectionCumulusId,
+        published: true,
+        cmr_link: 'https://cmr.uat.earthdata.nasa.gov/search/granules.json?concept_id=G1224390942-PO_NGP_UAT',
```

Comment:
No real issues with the URI, but should we make it a more obviously fake concept_id?

Reply:
Sure

https://github.com/nasa/cumulus/pull/2415/commits/cf36176ce0524660eccc651caf98b45a650457e0

I like having test data look a _lot_ like real data but there's an easy middle ground.